denizen_scripts/HeroCraft/missions/PvE/mission_smelt.dsc

```diff
@@ -21,6 +21,10 @@ mission_smelt:
       - 32
       - 48
       - 64
+    steel_ingot:
```

Comment:
These values should be in line with the other values

Reply:
I thought that with steel being harder to smelt, it shouldn't take as many.